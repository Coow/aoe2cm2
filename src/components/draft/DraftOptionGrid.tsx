import * as React from 'react';
import DraftOptionPanel from "../../containers/DraftOptionPanel";
import DraftOptionPanelType from "../../constants/DraftOptionPanelType";
import DraftOption from "../../models/DraftOption";
import i18next from "i18next";
import i18n from "i18next";

interface IProps {
    draftOptions: DraftOption[]
}

class DraftOptionGrid extends React.Component<IProps, object> {
    public render() {

        const panels = this.props.draftOptions.sort(this.compareDraftOptions).map((draftOption, index) => {
            return React.createElement(DraftOptionPanel, {
                draftOptionPanelType: DraftOptionPanelType.CHOICE,
                active: false,
                draftOption: draftOption,
                key: index,
                highlighted: false,
            });
        });

        const randomOption = React.createElement(DraftOptionPanel, {
            draftOptionPanelType: DraftOptionPanelType.CHOICE,
            active: false,
            draftOption: DraftOption.RANDOM,
            key: -1,
            highlighted: false,
        });

        return (
            <div id="civgrid" className="chooser">
                <div className="chooser-grid">
                    {randomOption}
                    {panels}
                </div>
            </div>
        );
    }

    private compareDraftOptions(a: DraftOption, b: DraftOption) {
        const aName = i18next.t(['civs.' + a.name, a.name]);
        const bName = i18next.t(['civs.' + b.name, b.name]);
        return aName.localeCompare(bName, i18n.language);
    }
}

export default DraftOptionGrid;
