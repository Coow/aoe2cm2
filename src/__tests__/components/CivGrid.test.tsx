import {shallow} from "enzyme";
import DraftOptionGrid from "../../components/draft/DraftOptionGrid";
import Civilisation from "../../models/Civilisation";

it('DraftOptionGrid renders correctly', () => {
    const component = shallow(<DraftOptionGrid draftOptions={Civilisation.ALL}/>);
    expect(component).toMatchSnapshot();
});
