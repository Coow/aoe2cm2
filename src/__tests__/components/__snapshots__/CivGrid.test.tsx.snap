// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DraftOptionGrid renders correctly 1`] = `
<div
  className="chooser"
  id="civgrid"
>
  <div
    className="chooser-grid"
  >
    <Connect(Component)
      active={false}
      draftOption={
        DraftOption {
          "i18nPrefix": "civs.",
          "id": "RANDOM",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/random-left.apng",
            "animated_right": "/images/units-animated/random-right.apng",
            "emblem": "/images/civemblems/random.png",
            "unit": "/images/civs/random.png",
          },
          "name": "RANDOM",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="-1"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 2,
          "i18nPrefix": "civs.",
          "id": "Aztecs",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/aztecs-left.apng",
            "animated_right": "/images/units-animated/aztecs-right.apng",
            "emblem": "/images/civemblems/aztecs.png",
            "unit": "/images/civs/aztecs.png",
          },
          "name": "Aztecs",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="0"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 4,
          "i18nPrefix": "civs.",
          "id": "Berbers",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/berbers-left.apng",
            "animated_right": "/images/units-animated/berbers-right.apng",
            "emblem": "/images/civemblems/berbers.png",
            "unit": "/images/civs/berbers.png",
          },
          "name": "Berbers",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="1"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Britons",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/britons-left.apng",
            "animated_right": "/images/units-animated/britons-right.apng",
            "emblem": "/images/civemblems/britons.png",
            "unit": "/images/civs/britons.png",
          },
          "name": "Britons",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="2"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 5,
          "i18nPrefix": "civs.",
          "id": "Burmese",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/burmese-left.apng",
            "animated_right": "/images/units-animated/burmese-right.apng",
            "emblem": "/images/civemblems/burmese.png",
            "unit": "/images/civs/burmese.png",
          },
          "name": "Burmese",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="3"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Byzantines",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/byzantines-left.apng",
            "animated_right": "/images/units-animated/byzantines-right.apng",
            "emblem": "/images/civemblems/byzantines.png",
            "unit": "/images/civs/byzantines.png",
          },
          "name": "Byzantines",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="4"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Celts",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/celts-left.apng",
            "animated_right": "/images/units-animated/celts-right.apng",
            "emblem": "/images/civemblems/celts.png",
            "unit": "/images/civs/celts.png",
          },
          "name": "Celts",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="5"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Chinese",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/chinese-left.apng",
            "animated_right": "/images/units-animated/chinese-right.apng",
            "emblem": "/images/civemblems/chinese.png",
            "unit": "/images/civs/chinese.png",
          },
          "name": "Chinese",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="6"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 4,
          "i18nPrefix": "civs.",
          "id": "Ethiopians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/ethiopians-left.apng",
            "animated_right": "/images/units-animated/ethiopians-right.apng",
            "emblem": "/images/civemblems/ethiopians.png",
            "unit": "/images/civs/ethiopians.png",
          },
          "name": "Ethiopians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="7"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Franks",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/franks-left.apng",
            "animated_right": "/images/units-animated/franks-right.apng",
            "emblem": "/images/civemblems/franks.png",
            "unit": "/images/civs/franks.png",
          },
          "name": "Franks",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="8"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Goths",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/goths-left.apng",
            "animated_right": "/images/units-animated/goths-right.apng",
            "emblem": "/images/civemblems/goths.png",
            "unit": "/images/civs/goths.png",
          },
          "name": "Goths",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="9"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 2,
          "i18nPrefix": "civs.",
          "id": "Huns",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/huns-left.apng",
            "animated_right": "/images/units-animated/huns-right.apng",
            "emblem": "/images/civemblems/huns.png",
            "unit": "/images/civs/huns.png",
          },
          "name": "Huns",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="10"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 3,
          "i18nPrefix": "civs.",
          "id": "Incas",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/incas-left.apng",
            "animated_right": "/images/units-animated/incas-right.apng",
            "emblem": "/images/civemblems/incas.png",
            "unit": "/images/civs/incas.png",
          },
          "name": "Incas",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="11"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 3,
          "i18nPrefix": "civs.",
          "id": "Indians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/indians-left.apng",
            "animated_right": "/images/units-animated/indians-right.apng",
            "emblem": "/images/civemblems/indians.png",
            "unit": "/images/civs/indians.png",
          },
          "name": "Indians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="12"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 3,
          "i18nPrefix": "civs.",
          "id": "Italians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/italians-left.apng",
            "animated_right": "/images/units-animated/italians-right.apng",
            "emblem": "/images/civemblems/italians.png",
            "unit": "/images/civs/italians.png",
          },
          "name": "Italians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="13"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Japanese",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/japanese-left.apng",
            "animated_right": "/images/units-animated/japanese-right.apng",
            "emblem": "/images/civemblems/japanese.png",
            "unit": "/images/civs/japanese.png",
          },
          "name": "Japanese",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="14"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 5,
          "i18nPrefix": "civs.",
          "id": "Khmer",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/khmer-left.apng",
            "animated_right": "/images/units-animated/khmer-right.apng",
            "emblem": "/images/civemblems/khmer.png",
            "unit": "/images/civs/khmer.png",
          },
          "name": "Khmer",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="15"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 2,
          "i18nPrefix": "civs.",
          "id": "Koreans",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/koreans-left.apng",
            "animated_right": "/images/units-animated/koreans-right.apng",
            "emblem": "/images/civemblems/koreans.png",
            "unit": "/images/civs/koreans.png",
          },
          "name": "Koreans",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="16"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 3,
          "i18nPrefix": "civs.",
          "id": "Magyars",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/magyars-left.apng",
            "animated_right": "/images/units-animated/magyars-right.apng",
            "emblem": "/images/civemblems/magyars.png",
            "unit": "/images/civs/magyars.png",
          },
          "name": "Magyars",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="17"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 5,
          "i18nPrefix": "civs.",
          "id": "Malay",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/malay-left.apng",
            "animated_right": "/images/units-animated/malay-right.apng",
            "emblem": "/images/civemblems/malay.png",
            "unit": "/images/civs/malay.png",
          },
          "name": "Malay",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="18"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 4,
          "i18nPrefix": "civs.",
          "id": "Malians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/malians-left.apng",
            "animated_right": "/images/units-animated/malians-right.apng",
            "emblem": "/images/civemblems/malians.png",
            "unit": "/images/civs/malians.png",
          },
          "name": "Malians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="19"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 2,
          "i18nPrefix": "civs.",
          "id": "Mayans",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/mayans-left.apng",
            "animated_right": "/images/units-animated/mayans-right.apng",
            "emblem": "/images/civemblems/mayans.png",
            "unit": "/images/civs/mayans.png",
          },
          "name": "Mayans",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="20"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Mongols",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/mongols-left.apng",
            "animated_right": "/images/units-animated/mongols-right.apng",
            "emblem": "/images/civemblems/mongols.png",
            "unit": "/images/civs/mongols.png",
          },
          "name": "Mongols",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="21"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Persians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/persians-left.apng",
            "animated_right": "/images/units-animated/persians-right.apng",
            "emblem": "/images/civemblems/persians.png",
            "unit": "/images/civs/persians.png",
          },
          "name": "Persians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="22"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 4,
          "i18nPrefix": "civs.",
          "id": "Portuguese",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/portuguese-left.apng",
            "animated_right": "/images/units-animated/portuguese-right.apng",
            "emblem": "/images/civemblems/portuguese.png",
            "unit": "/images/civs/portuguese.png",
          },
          "name": "Portuguese",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="23"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Saracens",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/saracens-left.apng",
            "animated_right": "/images/units-animated/saracens-right.apng",
            "emblem": "/images/civemblems/saracens.png",
            "unit": "/images/civs/saracens.png",
          },
          "name": "Saracens",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="24"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 3,
          "i18nPrefix": "civs.",
          "id": "Slavs",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/slavs-left.apng",
            "animated_right": "/images/units-animated/slavs-right.apng",
            "emblem": "/images/civemblems/slavs.png",
            "unit": "/images/civs/slavs.png",
          },
          "name": "Slavs",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="25"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 2,
          "i18nPrefix": "civs.",
          "id": "Spanish",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/spanish-left.apng",
            "animated_right": "/images/units-animated/spanish-right.apng",
            "emblem": "/images/civemblems/spanish.png",
            "unit": "/images/civs/spanish.png",
          },
          "name": "Spanish",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="26"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Teutons",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/teutons-left.apng",
            "animated_right": "/images/units-animated/teutons-right.apng",
            "emblem": "/images/civemblems/teutons.png",
            "unit": "/images/civs/teutons.png",
          },
          "name": "Teutons",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="27"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Turks",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/turks-left.apng",
            "animated_right": "/images/units-animated/turks-right.apng",
            "emblem": "/images/civemblems/turks.png",
            "unit": "/images/civs/turks.png",
          },
          "name": "Turks",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="28"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 5,
          "i18nPrefix": "civs.",
          "id": "Vietnamese",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/vietnamese-left.apng",
            "animated_right": "/images/units-animated/vietnamese-right.apng",
            "emblem": "/images/civemblems/vietnamese.png",
            "unit": "/images/civs/vietnamese.png",
          },
          "name": "Vietnamese",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="29"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 1,
          "i18nPrefix": "civs.",
          "id": "Vikings",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/vikings-left.apng",
            "animated_right": "/images/units-animated/vikings-right.apng",
            "emblem": "/images/civemblems/vikings.png",
            "unit": "/images/civs/vikings.png",
          },
          "name": "Vikings",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="30"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 6,
          "i18nPrefix": "civs.",
          "id": "Bulgarians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/bulgarians-left.apng",
            "animated_right": "/images/units-animated/bulgarians-right.apng",
            "emblem": "/images/civemblems/bulgarians.png",
            "unit": "/images/civs/bulgarians.png",
          },
          "name": "Bulgarians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="31"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 6,
          "i18nPrefix": "civs.",
          "id": "Cumans",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/cumans-left.apng",
            "animated_right": "/images/units-animated/cumans-right.apng",
            "emblem": "/images/civemblems/cumans.png",
            "unit": "/images/civs/cumans.png",
          },
          "name": "Cumans",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="32"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 6,
          "i18nPrefix": "civs.",
          "id": "Lithuanians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/lithuanians-left.apng",
            "animated_right": "/images/units-animated/lithuanians-right.apng",
            "emblem": "/images/civemblems/lithuanians.png",
            "unit": "/images/civs/lithuanians.png",
          },
          "name": "Lithuanians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="33"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 6,
          "i18nPrefix": "civs.",
          "id": "Tatars",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/tatars-left.apng",
            "animated_right": "/images/units-animated/tatars-right.apng",
            "emblem": "/images/civemblems/tatars.png",
            "unit": "/images/civs/tatars.png",
          },
          "name": "Tatars",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="34"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 7,
          "i18nPrefix": "civs.",
          "id": "Burgundians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/burgundians-left.apng",
            "animated_right": "/images/units-animated/burgundians-right.apng",
            "emblem": "/images/civemblems/burgundians.png",
            "unit": "/images/civs/burgundians.png",
          },
          "name": "Burgundians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="35"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 7,
          "i18nPrefix": "civs.",
          "id": "Sicilians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/sicilians-left.apng",
            "animated_right": "/images/units-animated/sicilians-right.apng",
            "emblem": "/images/civemblems/sicilians.png",
            "unit": "/images/civs/sicilians.png",
          },
          "name": "Sicilians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="36"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 8,
          "i18nPrefix": "civs.",
          "id": "Bohemians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/bohemians-left.apng",
            "animated_right": "/images/units-animated/bohemians-right.apng",
            "emblem": "/images/civemblems/bohemians.png",
            "unit": "/images/civs/bohemians.png",
          },
          "name": "Bohemians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="37"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 8,
          "i18nPrefix": "civs.",
          "id": "Poles",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/poles-left.apng",
            "animated_right": "/images/units-animated/poles-right.apng",
            "emblem": "/images/civemblems/poles.png",
            "unit": "/images/civs/poles.png",
          },
          "name": "Poles",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="38"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 9,
          "i18nPrefix": "civs.",
          "id": "Bengalis",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/bengalis-left.apng",
            "animated_right": "/images/units-animated/bengalis-right.apng",
            "emblem": "/images/civemblems/bengalis.png",
            "unit": "/images/civs/bengalis.png",
          },
          "name": "Bengalis",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="39"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 9,
          "i18nPrefix": "civs.",
          "id": "Dravidians",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/dravidians-left.apng",
            "animated_right": "/images/units-animated/dravidians-right.apng",
            "emblem": "/images/civemblems/dravidians.png",
            "unit": "/images/civs/dravidians.png",
          },
          "name": "Dravidians",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="40"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 9,
          "i18nPrefix": "civs.",
          "id": "Gurjaras",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/gurjaras-left.apng",
            "animated_right": "/images/units-animated/gurjaras-right.apng",
            "emblem": "/images/civemblems/gurjaras.png",
            "unit": "/images/civs/gurjaras.png",
          },
          "name": "Gurjaras",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="41"
    />
    <Connect(Component)
      active={false}
      draftOption={
        Civilisation {
          "gameVersion": 9,
          "i18nPrefix": "civs.",
          "id": "Hindustanis",
          "imageUrls": Object {
            "animated_left": "/images/units-animated/hindustanis-left.apng",
            "animated_right": "/images/units-animated/hindustanis-right.apng",
            "emblem": "/images/civemblems/hindustanis.png",
            "unit": "/images/civs/hindustanis.png",
          },
          "name": "Hindustanis",
        }
      }
      draftOptionPanelType="choice"
      highlighted={false}
      key="42"
    />
  </div>
</div>
`;
